az-account: add page (#6204)
tldr-pages/tldr: pages/common/az-account.md

# az account

> > Manage Azure subscription information.

Author: ```suggestion
> Manage Azure subscription information.
> Part of `az`, the command-line client for Microsoft Azure.
```

> More information: <https://docs.microsoft.com/cli/azure/account>.

> - Get a list of subscriptions for the logged in account:

Reviewer: ```suggestion
- Print a list of subscriptions for the logged in account:
```

`az account list`

> - Set a `subscription` to be the current active subscription:

Reviewer 2: ```suggestion
- Set a `subscription` to be the currently active subscription:
```
Missed that one

> - Set a `subscription` to be the currently active subscription:

Reviewer: ```suggestion
- Set a `subscription` to be the active subscription:
```

Reviewer: This would need to be applied throughout if agreed on

Reviewer 2: I think both are fine, but maybe with `currently` it reads a bit more fluently.

Author: > This would need to be applied throughout if agreed on

Yep, let me send in a PR for the one that we have merged already :)

Author: IMO `currently` reads better .. for now going with that :)

> `az account set --subscription {{00000000-0000-0000-0000-000000000000}}`

Reviewer: ```suggestion
`az account set --subscription {{subscription_id}}`
```

I don't know if the language is correct but that string of numbers cannot be used

> - List supported regions for the current active subscription:

Reviewer 2: ```suggestion
- List supported regions for the currently active subscription:
```

`az account list-locations`

> - Print an access token to use with `MS Graph API`:

Reviewer 2: ```suggestion
- Print an access token to be used with `MS Graph API`:
```
I'm not sure if this is really correct

> - Get an access token to use with `MS Graph API`:

Reviewer: ```suggestion
- Print an access token to use with `MS Graph API`:
```

`az account get-access-token --resource-type {{ms-graph}}`

> - Print details of the current active subscription in specified format:

Reviewer 2: ```suggestion
- Print details of the currently active subscription in the specified format:
```

> - Get details of current active subscription in `json` format:

Reviewer: ```suggestion
- Print details of the current active subscription in `JSON` format:
```

> - Print details of the currently active subscription in the specified format:

Reviewer: ```suggestion
- Print details of the currently active subscription in a specific format:
```

Reviewer 2: `the specific` is what we use in most pages, even tough a little variety doesn't hurt.

Reviewer: I still feel that `a` is required, and with an `a`, `specific` reads nicer

> `az account show --output json`

Reviewer: Does this support other types of output formats, if it does, we should list them {{1|2|3}}

Author: ```suggestion
`az account show --output {{json|tsv|table|yaml}}`
```
